link_shorterner/test/shortener_shortener_SUITE.erl

```diff
@@ -0,0 +1,70 @@
+-module(shortener_shortener_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+
+-export([all/0,
+         init_per_testcase/2,
+         end_per_testcase/2]).
+-export([test_notfound/1,
+         test_created/1,
+         test_ok/1]).
+
+all() ->
+    [test_notfound,
+     test_created,
+     test_ok].
+
+init_per_testcase(_,Config) ->
```

Comment:
I think it's enough to start the app in the init_per_suite, this could get really slow on a bigger app with more tests.